doc/how.rst

```diff
@@ -29,10 +29,14 @@ A objective should define a `set_data` method that allows
 to specify the data and a `compute` method that allows
 to evaluate the objective for a given value of the iterate.
 
+Multiple values can be computed in one objective as long as they are
+stored in a dictionary with a key being `objective_value`. This allows
+to compare different metrics in the same run.
```

Comment:
to compute different metrics in a single run ?

Reply:
With multiple outputs, we can compute multiple metrics when the function compute from the objective is called.
So in theory run the command once and compute each value for the different metrics needed.

Reply:
my beef is with "compare" (a comparison should be between solvers, and then multiple runs are needed) but that's cosmetic

Reply:
I can use the words "to monitor" used in the original PR if you prefer.